### ISSUE TITLE
ADMIN - REVIEW REPORT: Create U CODE list for HC cases

### ISSUE THREAD

**Issue by IlseFerris** [Hennepin-County/MAXIS-scripts#1322]
Will need to review if U Coded Health Care cases auto-close, and then if so then review the need to create a worklist.

**Comment by IlseFerris** [comment]
I will be contacting the BI team to see how we can get ALL U codes into ES workflow, hopefully for the person who had the assignment last.

**Comment by IlseFerris** [comment]
BI team is going to add this to ES Workflow in the sprint starting 07/11/2023. Recommending we won't fix at this point.

**Comment by C-Love** [comment]
Project exists for assigning U Code cases through ES Workflow or other possibilities. We can work through this issue once this assignment is in place.

**Comment by IlseFerris** [comment]
DaNM: can we close this as a won't fix since we're building the main supports into ELIG SUMMARY?

**Comment by IlseFerris** [comment]
I reviewed approximately 50 cases, and I was able to confirm that the U code, if present for any HH member on the HC REVW POP UP, the case's HC renewal status will be a U. It overrides all the other person-based renewal statuses.

**Comment by IlseFerris** [comment]
Closing as a won't fix. Current MAXIS Health Care expansion does not have current discrepancy tasks. If U Codes are needed, they can be captured by running REPT/REVS and/or REPT/REVW.
